server/templates/static/homepage.html

```diff
@@ -49,8 +49,8 @@ <h2>{% trans %}What's new{% endtrans %}</h2>
       <dt>{{ blog_date }}</dt>
       <dd>
         {# TRANSLATORS: Blog post entry. Please take care to maintain valid <a> tags in translation. See this string in action at https://datacommons.org #}
-        {% trans blog_url="https://docs.datacommons.org/2021/06/01/updates.html",project="Data Commons" -%}
-        We have launched exciting new features, including internationalization support, a new graph browser and scatter plot explorer, new Stats API, updated documentation, and new data. <a href={{blog_url}}>Read our blog post for more details</a>.
+        {% trans blog_url="https://docs.datacommons.org/2020/07/26/updates.html",project="Data Commons" -%}
+        We have launched exciting new features, including a new map explorer, a new statistical variable menu, and new data. <a href={{blog_url}}>Read our blog post for more details</a>.
```

Comment:
maybe clarify that the sv menu is launched across our viz tools?

Reply:
done